data/forms/create-new-person.jsx

```diff
@@ -66,6 +69,20 @@ export default {
           label: 'Nationality',
           rules: { required: true },
         },
+        {
+          component: 'Select',
+          name: 'ethnicities',
+          label: 'Ethnicity',
+          options: Object.keys(ETHNICITIES),
+        },
+        {
+          component: 'Select',
+          name: 'Sub-ethnicity',
```

Comment:
This should be `ethnicity` because we are only interested in the `id` returned by the the sub-ethnicity select.
We can call the other one `macro_ethnicity`, or something like that.